INPTP_AppForFixing/EmployeeDialog.cs

```diff
@@ -9,22 +8,16 @@ public partial class EmployeeDialog : Form
         private MainForm main;
         private bool employee = false;
         private Boss bossForEmployee;
+        private Employee emp;
```

Comment:
no abbreviations - attribute name should be employee

Reply:
fixed